EIPS/eip-TBD.md

```diff
@@ -1,85 +1,57 @@
+For every ERC-721 transfer, *except* for initial mint, the set of the corresponding `tokenId` MUST be appended with the previous owner (i.e. the transfer's `from` address).
 
-Royalty payment by the token owner MUST be accepted by the token contract via a dedicated function, `TBD()`.
+Royalty payment by the new token owner MUST be accepted by the token contract via a dedicated function, `TBD()`.
 Upon receipt of royalties, the take-back set MUST be emptied (i.e. transfer permissions collapse back to standard ERC-721 rules).
-After each token transfer, there SHOULD be a grace period during which the token is frozen and take-backs are disabled to allow the new owner to decide whether to pay the royalty or allow the take-back set to grow.
+After each token transfer, there SHOULD be a grace period during which the token is frozen and take-backs are disabled to allow the new owner to decide whether to either pay the royalty or allow the take-back set to grow.
 
 In addition to clearing the take-back set, receipt of royalties MUST begin a temporary window during which *any* address MAY purchase the token for a price that is a function of the royalty received (which can be viewed as the inverse of the `royaltyInfo()` function of ERC-2981).
 This process is referred to as *auto-listing* and is not limited to addresses in the take-back set.
-The token MUST NOT be transferrable for the duration of the *auto-listing*.
+The token MUST NOT be transferrable for the duration of the auto-listing.
 Implementations MAY specify the duration of the auto-listing window at their discretion but it MUST be finite and SHOULD be long enough to allow arbitrage opportunities to be detected and, after expiration, transfer permissions MUST collapse back to standard ERC-721 rules.
 
+In the event of a sale due to an auto-listing, (a) the corresponding royalties MUST be reimbursed, (b) the auto-listing price MUST be transferred to the token owner (i.e. the same address receives both the royalties and the sale price), and (c) the token MUST be transferred to the address paying the price.
+The new owner MUST be subject to the same mechanism rules; i.e. take-back, royalty payment, and auto-listing.
+If no sale occurs during the auto-listing window, the contract SHOULD transfer the royalty payment to the NFT creator.
+
+Marketplaces facilitating sales SHOULD forward royalties in the same transaction as the sale to guarantee payment within the grace period.
+Marketplaces MAY allow buyers to configure either the auto-listing price or the royalties paid (as they are equivalent), or they MAY provide sensible default values at their discretion.
+If allowing buyers to configure values, marketplaces SHOULD limit this to the auto-listing price for ease of understanding as royalties are effectively calculated on this value.
```

Comment:
I'd maybe remove this or change to RECOMMEND, since we can't foresee what they will come up with

Reply:
RECOMMENDED has the same definition as SHOULD